src/components/DropDown/index.scss

```diff
@@ -96,12 +93,11 @@
     position: absolute;
     top: 50%;
     right: 10px;
-    margin-top: -9px;
+    margin-top: -8px; // Flag icon (height/2)
```

Comment:
This should be styled in a way that any user with icons can use this dropdown (so not specific to kununu footer). One way to do this would be to make the text and icon span inline-block so that you can vertical align center and then add float right to the icon span and remove the padding-right: 40px from the li wrapper. 

I think doing it this way would make it more flexible and usable for diff situations.